### ISSUE TITLE
Failed to activate the vue-hyperclick package

### ISSUE THREAD

**Issue by rodneyrehm** [monterail/atom-vue-hyperclick#2]
[Enter steps to reproduce below:]

1. start atom with the plugin installed

**Atom Version**: 1.12.7
**Electron Version**: 1.3.13
**System**: Mac OS X 10.10.5
**Thrown From**: [vue-hyperclick](https://github.com/monterail/atom-vue-hyperclick) package, v0.1.0

### Stack Trace

Failed to activate the vue-hyperclick package

```
At Cannot find module '../../js-hyperclick/lib/suggestions.js'

Error: Cannot find module '../../js-hyperclick/lib/suggestions.js'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at makeProvider (/Users/rodneyrehm/.atom/packages/vue-hyperclick/lib/vue-hyperclick.js:9:50)
    at Object.getProvider (/Users/rodneyrehm/.atom/packages/vue-hyperclick/lib/vue-hyperclick.js:29:20)
    at Package.module.exports.Package.activateServices (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:359:69)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:187:16)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:157:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:150:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:550:34)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:531:29
    at Config.module.exports.Config.transactAsync (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:526:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:508:46)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:792:28
```

### Commands

```
```

### Config

```json
{
  "core": {
    "telemetryConsent": "limited"
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.29.14 (inactive)
autocomplete-modules, v1.6.7 (inactive)
change-case, v0.6.3 (inactive)
editorconfig, v2.0.5 (inactive)
es-identifier-highlight, v0.4.3 (inactive)
expand-region, v0.2.6 (inactive)
file-icons, v1.7.25 (inactive)
git-blame, v0.4.11 (inactive)
git-diff-popup, v0.1.8 (inactive)
git-plus, v6.0.1 (inactive)
git-time-machine, v1.5.4 (inactive)
hyperclick, v0.0.39 (inactive)
js-hyperclick, v1.10.0 (inactive)
language-gettext, v0.6.1 (inactive)
language-vue, v0.21.0 (inactive)
linter, v1.11.18 (inactive)
linter-eslint, v8.0.0 (inactive)
linter-jsonlint, v1.3.0 (inactive)
minimap, v4.25.7 (inactive)
minimap-bookmarks, v0.4.2 (inactive)
minimap-cursorline, v0.2.0 (inactive)
minimap-find-and-replace, v4.5.1 (inactive)
string-encoder, v0.1.0 (inactive)
text-manipulation, v0.6.0 (inactive)
vue-hyperclick, v0.1.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (inactive)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (inactive)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (inactive)
archive-view, v0.62.0 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.13.1 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.4 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.42.0 (inactive)
bracket-matcher, v0.82.2 (inactive)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.202.2 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.60.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (inactive)
metrics, v1.1.2 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (inactive)
snippets, v1.0.4 (inactive)
spell-check, v0.68.4 (inactive)
status-bar, v1.4.1 (inactive)
styleguide, v0.47.2 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.0 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.210.0 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (inactive)
wrap-guide, v0.38.2 (inactive)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.0 (inactive)
language-csharp, v0.13.0 (inactive)
language-css, v0.40.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.43.0 (inactive)
language-html, v0.47.1 (inactive)
language-hyperlink, v0.16.1 (inactive)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (inactive)
language-json, v0.18.3 (inactive)
language-less, v0.29.6 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (inactive)
language-php, v0.37.3 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.1 (inactive)
language-ruby, v0.70.2 (inactive)
language-ruby-on-rails, v0.25.1 (inactive)
language-sass, v0.57.0 (inactive)
language-shellscript, v0.23.0 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.25.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.29.1 (inactive)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (inactive)
language-yaml, v0.27.1 (inactive)

# Dev
No dev packages
```

**Comment by orenmizr** [comment]
fails after the js hyperclick update 1.10.0

**Comment by stealthybox** [comment]
Confirming this:
- Windows 7
- Atom 1.12.7
- js-hyperclick@1.10.0

Downgrading to **1.9.0** fixes the issue for now:
`apm deinstall js-hyperclick && apm install js-hyperclick@1.9.0`

**Comment by Frizi** [comment]
fixed in 0.2.0, should work with both versions.
